### PR TITLE
Proposal for Chaincode Analyzer

### PR DESCRIPTION
**Pull request by kzhry**
# Short Description
A static tool to check and detect potentially risky codes in smart contracts (a.k.a, chaincodes).

Signed-off-by: kzhry <y-kazuhiro@fujitsu.com>

### CODE REVIEW

**Comment by tkuhrt** [comment]
Looks like the commit did not have DCO. Can you fix that?

**Comment by kzhry** [comment]
Hi Tracy,

Thank you for your suggestion. 
I fixed the commit and the PR comment for consistency.

File: labs/chaincode-analyzer.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+
+# Scope of Lab
+The aim of Chaincode Analyzer is to provide the way to check whether the smart contracts includes potentially risky codes or not.
+The tool performs static analysis on chaincode source codes written in Golang and detects potentially risky parts such as generating random values, accessing outside of blockchain networks and iteration on map object.
+For more details of the tool and target risks:
+https://github.com/FujitsuLaboratories/ChaincodeAnalyzer
+
+In Hyperledger Fabric, developers can use general purpose programming languages, such as JavaScript, Golang and Java, to implement their chaincodes.
+Compared to using domain specific languages (DSLs) like Solidity of Ethereum, the efforts of implementing smart contracts are smaller since develpers do not need to study new language for smart contracts.
```
Comment (tkuhrt) [inline labs/chaincode-analyzer.md]: Typo develpers->developers

**Comment by hartm** [comment]
> LGTM with the one change noted above.
> 
> @hartm can you confirm sponsorship?

Confirmed.  Thanks Tracy!

File: labs/chaincode-analyzer.md (hunk 1 above)
Comment (kzhry) [reply labs/chaincode-analyzer.md]: Thank you for your review. I fixed the typos at e451d55.

**Comment by kzhry** [comment]
Hi @tkuhrt, @trbs, @vipinsun and @lehors,

I'm just wondering that the reason of the pending status.
If there is any problem that I should work on (solving the DCO problem above?), could you please let me know? 

Thanks,
kzhry

**Comment by trbs** [comment]
@tkuhrt could it be the DCO bot is stuck ?

image

**Pull request closed without merging**